pkg/tags/tags_test.go

```diff
@@ -121,7 +128,7 @@ func TestListAll(t *testing.T) {
 	}
 
 	if len(tagList3) != 5 {
-		t.Fatalf("want %d tags but got %d", 5, len(tagList2))
+		t.Fatalf("want %d tags but got %d", 5, len(tagList3))
```

Comment:
how did this change?

Reply:
Variable logged didn't match one in preceding check, which came to light when the test failed with "want 0, got 0" during some earlier state.